client/app/services/event-notifications.service.js

```diff
@@ -42,21 +42,73 @@
       }
     };
 
+    function miqFormatNotification(text, bindings) {
+      var str = __(text);
+      lodash.each(bindings, function (value, key) {
+        str = str.replace(new RegExp('%{' + key + '}', 'g'), value.text);
+      });
+
+      return str;
+    }
+
     doReset();
 
+    var cable = ActionCable.createConsumer('/ws/notifications');
+
+    cable.subscriptions.create('NotificationChannel', {
+      disconnected: function () {
+        // eslint-disable-next-line consistent-this
```

Comment:
@AllenBW thoughts here?  I see this alot, and is this something we want to add to the .eslintrc.json or get people to fix it?